packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
@pablonete I think my main question is what would it take to simply use Tooltip (not host) and hide/show it using your own logic. The idea of TooltipHost was to add a simple, base case experience for those that just needed a simple tooltip. Custom scenarios were expected to be created using the tooltip itself.